src/inference/include/openvino/runtime/intel_gna/properties.hpp

```diff
@@ -117,22 +117,37 @@ inline std::istream& operator>>(std::istream& is, ExecutionMode& execution_mode)
  */
 enum class HWGeneration {
     UNDEFINED = 0,  //!< GNA HW generation is undefined
-    GNA_2_0 = 1,    //!< GNA HW generation 2.0
-    GNA_3_0 = 2,    //!< GNA HW generation 3.0
-    GNA_3_5 = 3,    //!< GNA HW generation 3.5
+    GNA_1_0 = 1,    //!< GNA HW generation 1.0
```

Comment:
Could it lead to binary incompatibility?

Reply:
I will leave the old enum values to be sure there's no issue.